e2e/web/specs/play-sudoku.spec.ts

```diff
@@ -242,7 +250,7 @@ test.describe("undo", () => {
   }) => {
     const sudokuBoard = new SudokuBoardComponent(resumeGame);
     await sudokuBoard.cell[7][7].click();
-    await sudokuBoard.note.click();
+    await sudokuBoard.page.keyboard.press("T");
```

Comment:
Could we have separate test for toggle note functionality?

Reply:
Done!